test/test_pdo.py

```diff
@@ -1,41 +1,51 @@
 import os.path
 import unittest
+
 import canopen
+try:
+    import canmatrix
+except ImportError:
+    canmatrix = None
```

Comment:
Here again: Let's just `pass` on the import error and skip the test when access to any needed names in `canmatrix` gives a `NameError`.